### PR TITLE
Petprofile

### PR DESCRIPTION
**Pull request by nikkypyra**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
-        </a>
-      </header>
-    </div>
-  );
+    <>
+      <GlobalStyles />
+      <Switch>
+        <Route exact path="/pet-profile">
+          <PetProfile />
+        </Route>
```
Comment (carolinstuhr) [inline src/App.js:13]: Is the pet profile currently the only page in your app? In that case you probably wouldn't need a Router just yet. But you probably included it for future stories?

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { createGlobalStyle } from 'styled-components/macro'
+
+export default createGlobalStyle`
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
```
Comment (carolinstuhr) [inline src/GlobalStyles.js:7]: You are now giving every element margin and padding of 0, is that what you had planned or was it meant to be for the body?

**Review by carolinstuhr — COMMENTED** [review]
Looks good to me. Just 4 little comments that you maybe could take a look at.

File: src/components/TasksStyled.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+/*
+import { loadFromLocal, saveToLocal } from '../services'  
+
+const [checked, setChecked] = useState(loadFromLocal('checked'))
+
+  useEffect(() => {
+    saveToLocal('checked', checked)
+  }, [checked])
+*/
```
[the remark this answers is not in the document]
Comment (nikkypyra) [reply src/components/TasksStyled.js:96]: I don't need it at the moment, but I will need it for my next step so I saved it for now.

File: src/components/PetProfile.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react'
+import ProfileHeader from './ProfileHeader'
+import TasksStyled from './TasksStyled'
+import tasks from '../tasks.json'
+
+export default function PetProfile() {
+  const [todos, setTodos] = useState(tasks)
+  console.log(tasks)
+
```
[the remark this answers is not in the document]
Comment (nikkypyra) [reply src/components/PetProfile.js:9]: Thank you! It was for testing. Forgot to delete it.

File: src/App.js (hunk 1 above)
Comment (nikkypyra) [reply src/App.js:13]: Exactly. I was thinking ahead, but I probably should have waited until my next Sprint to add the router.

File: src/GlobalStyles.js (hunk 2 above)
Comment (nikkypyra) [reply src/GlobalStyles.js:7]: It was meant for the body. Thanks for catching that!

**Pull request merged**